plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1963,28 +1972,30 @@ producer_plugin_impl::start_block_result producer_plugin_impl::start_block() {
    } else if (_producers.find(scheduled_producer.producer_name) == _producers.end()) {
       _pending_block_mode = pending_block_mode::speculating;
    } else if (num_relevant_signatures == 0) {
-      fc_elog(_log, "Not producing block because I don't have any private keys relevant to authority: ${authority}",
-              ("authority", scheduled_producer.authority));
+      fc_elog(_log, "Not producing block because I don't have any private keys relevant to authority: ${authority}, block ${t}",
```

Comment:
Should it be `block ${t}` when t is a timestamp?

Reply:
There are multiple ways to refer to a block. block number, block id, block timestamp, block slot (just the slot number), etc. Since it is obviously a block timestamp, I don't see saying anything except `block`.

Reply:
Hum, maybe it is obvious to you, but I do think it would be clearer to state what the number is. 
Not asking to change anything in this PR though, more a philosophical discussion.

Reply:
It will look like this: `Not producing block because I don't have any private keys relevant to authority: ..., block 2024-08-28T01:08:31.500`